### PR TITLE
allow overwrite of the root postgres username

### PR DESCRIPTION
**Pull request by mmounirou**
The postgres docker image (unlike the mariadb) provide the ability to overwrite the default postgres's root user .

The aim of this merge request is to provide the same feature when using the wallabag image .

This is done by : 

 - [x] adding a new environment variable POSTGRES_USER
 - [x] making the ansible playbook using it instead of the hard coded "postgres"
 - [x] setting the default value for the environment variable to "postgres" to ensure backward compatibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +44,7 @@ $ docker run -v /opt/wallabag:/var/www/wallabag/data -p 80:80 wallabag/wallabag
 For using MariaDB or MySQL you have to define some environment variables with the container. Example:
 
 ```
-$ docker run docker run --name wallabag-db -e "MYSQL_ROOT_PASSWORD=my-secret-pw" -d mariadb
+$ docker run docker run --name wallabag-db -e "MYSQL_ROOT_PASSWORD=my-secret-pw"  -d mariadb
```
Comment (j0k3r) [inline README.md]: Not necessary

**Comment by j0k3r** [comment]
Good idea @mmounirou !

**Comment by xsteadfastx** [comment]
thats pretty cool. could you get rid of the extra space in line 47? and then i will merge it. :+1:

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**